app/assets/stylesheets/markets.scss

```diff
@@ -0,0 +1,142 @@
+// Place all the styles related to the markets controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+@import url('https://fonts.googleapis.com/css?family=Amatic+SC|Open+Sans');
+
+body {
+  font-family: 'Open Sans', sans-serif;
+  background-image: image-url("p-patch.jpg");
+  background-size: cover;
+  color: #004d00;
+}
+
+a:hover {text-decoration: underline;
+}
+
+a {
+  text-decoration:none;
+  color: #004d00;
+}
+
+nav {
+  text-align: center;
+  font-size: 16px;
+  background-color: #004d00;
+  border-radius: 5px;
+  padding: 5px;
+}
+
+.nav_link {
+  color: white;
+  font-family: 'Open Sans';
+}
+
+h1 {
+  font-family: 'Amatic SC', sans-serif;
+  text-align: center;
+  font-size: 3em;
+}
+
+div.fade {
+  width: 700px;
+  background-color: rgba(255, 255, 255, .75);
+  margin: 50px auto;
+  padding:  15px 50px;
+  border-radius: 10px;
+  text-align: left;
+}
+
+div.button {
+  display:inline-block;
+  padding: 40px;
+  border-radius: 10px;
+  border-width: medium;
+  margin: 10px;
+  margin-bottom: 20px;
+  text-align: center;
+
+}
+
+div.button > a {
+  text-decoration: none;
+  font-family: 'Amatic SC', sans-serif;
+  color: #ffffff;
+  font-weight: bold;
+  font-size: 2em;
+}
+
+div.button > a:hover {
+  font-size: 250%;
+}
+
+div.center{
+  text-align: center;
+}
+
+div.show_sales {
```

Comment:
For CSS styles that are multiple words, standards state that we should separate with the `-` rather than the `_`